x-pack/platform/plugins/shared/streams/server/lib/telemetry/ebt/client.ts

```diff
@@ -53,4 +61,8 @@ export class EbtTelemetryClient {
 
     this.analytics.reportEvent(STREAMS_STATE_ERROR_EVENT, errorData);
   }
+
+  public trackFeaturesIdentified(params: StreamsFeatureIdentificationIdentifiedProps) {
```

Comment:
out of scope but any reason we implemented the sig events telemetry in the browser instead of the server ?

Reply:
I think we just didn't think to deeply about it, and the existing telemetry was browser based so we just extended.
As we build the other two background tasks I think the telemetry will naturally move to the server (where possible).